### PR TITLE
added test and file with extended presence masks

### PR DESCRIPTION
**Pull request by quack-dev**

### CODE REVIEW

File: Test/PacketType/Ieee80211/RadioPacketTest.cs [hunk 1]
```diff
@@ -238,5 +238,19 @@ public void UnhandledRadioTapField()
...
+        public void ExtendedPresenceMaskFlags()
+        {
+            //Test for https://www.radiotap.org/#extended-presence-masks
+            var dev = new CaptureFileReaderDevice(NUnitSetupClass.CaptureDirectory + "80211_radiotap_with_extended_presence_mask.pcap");
+            dev.Open();
+            var rawCapture = dev.GetNextPacket();
+            dev.Close();
+            
+            var p = Packet.ParsePacket(rawCapture.LinkLayerType, rawCapture.Data) as RadioPacket;
+            var rate = ((RateRadioTapField)radioPacket[RadioTapType.Rate]).RateMbps;
```
Comment (chmorgan) [inline Test/PacketType/Ieee80211/RadioPacketTest.cs]: This line doesn't look correct. Does this test build for you?

**Comment by chmorgan** [comment]
Looks like a good start but the test isn't compiling. Wrong variable used?

File: Test/PacketType/Ieee80211/RadioPacketTest.cs (hunk 1 above)
Comment (quack-dev) [reply Test/PacketType/Ieee80211/RadioPacketTest.cs]: oops, I copied that line from somewhere else and totally forgot to change the variable name

**Comment by chmorgan** [comment]
@quack-dev mind taking a look at this branch, https://github.com/chmorgan/packetnet/tree/extendedp ? The radio tap handling was broken in a few different ways but should be all set now.

**Comment by Phyxion** [comment]
This can be closed now I think (@chmorgan / @quack-dev)?

**Comment by chmorgan** [comment]
I was waiting for @quack-dev to comment but I’m pretty sure the issue is
fixed now. I think this can be closed yes.

**Pull request closed without merging**